### PR TITLE
[OPIK-457] Fix Redis lock keys leak

### PR DESCRIPTION
**Pull request by thiagohora**
## Details
Radisson doesn't delete a lock key from Redis after releasing it by default. To tackle this issue, we are adding a new configuration, `ttlInSeconds`. This configuration will be used to set a ttl time for the lock every time a thread acquires a lock.

## Issues
OPIK-457

## Testing
- Check if a lock is evicted from Redis after ttl

### CODE REVIEW

**Review by idoberko2 — APPROVED** [review]
Great work!

File: apps/opik-backend/src/main/java/com/comet/opik/infrastructure/redis/RedissonLockService.java [hunk 1]
```diff
@@ -21,31 +24,54 @@ class RedissonLockService implements LockService {
     private final @NonNull RedissonReactiveClient redisClient;
     private final @NonNull DistributedLockConfig distributedLockConfig;
 
+    private record LockInstance(RPermitExpirableSemaphoreReactive semaphore, String locked) {
+
+        public void release() {
+            semaphore.release(locked)
+                    .subscribe(
+                            __ -> log.debug("Lock {} released successfully", locked),
+                            __ -> log.warn("Lock {} already released", locked));
```
Comment (andrescrz) [inline apps/opik-backend/src/main/java/com/comet/opik/infrastructure/redis/RedissonLockService.java:33]: Minor: wrap escaped values in the logs between single quotes.

File: apps/opik-backend/src/test/java/com/comet/opik/infrastructure/redis/RedissonLockServiceIntegrationTest.java [hunk 2]
```diff
@@ -3,29 +3,35 @@
...
+import java.time.Duration;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
 
+import static com.comet.opik.api.resources.utils.TestDropwizardAppExtensionUtils.AppContextConfig;
+import static com.comet.opik.api.resources.utils.TestDropwizardAppExtensionUtils.CustomConfig;
+import static com.comet.opik.api.resources.utils.TestDropwizardAppExtensionUtils.newTestDropwizardAppExtension;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
```
Comment (andrescrz) [inline apps/opik-backend/src/test/java/com/comet/opik/infrastructure/redis/RedissonLockServiceIntegrationTest.java:31]: Minor: please clean up all these JUnit assertions and move to assertJ.

**Review by andrescrz — APPROVED** [review]
Just two minor comments, the rest LGTM. Thanks for putting this together!

File: apps/opik-backend/src/main/java/com/comet/opik/infrastructure/redis/RedissonLockService.java (hunk 1 above)
Comment (thiagohora) [reply apps/opik-backend/src/main/java/com/comet/opik/infrastructure/redis/RedissonLockService.java:33]: Adding both in a next review

**Pull request merged**